pyproject.toml

```
@@ -8,7 +8,7 @@ authors = ["CodeForPoznań <hello@codeforpoznan.pl>"]
 Jinja2 = "^3.1.4"
 pydantic = "2.9.2"
 PySide6 = "6.8.3"
-python = "3.10.9"
+python = "~3.12"
```

as you're omitting patch version, you can just write 
```suggestion
python = "3.12"
```
and it will allow all patch versions of 3.12

---

```qtuser@f0311e9a9ef0:/app$ poetry add "python==3.12"
Skipping virtualenv creation, as specified in config file.

Could not find a matching version of package python

qtuser@f0311e9a9ef0:/app$ poetry add python@^3.12
Skipping virtualenv creation, as specified in config file.
```
Tilde version pin would result in the same behaviour (`>=3.12 <3.13`) and following commands fail. Altering `pyproject.toml` directly (`python = "^3.12”`, `python = “3.12”` or even `python = “^3.12.10”` or `python = “^3.12.10”` fails when running `poetry lock` afterwards. I am not a big fan of `poetry`. ;-)